brainscore/benchmarks/__init__.py

```diff
@@ -193,6 +193,8 @@ def _experimental_benchmark_pool():
     pool['dicarlo.SanghaviMurty2020.IT-pls'] = LazyLoad(DicarloSanghaviMurty2020ITPLS)
     from .rajalingham2020 import DicarloRajalingham2020ITPLS
     pool['dicarlo.Rajalingham2020.IT-pls'] = LazyLoad(DicarloRajalingham2020ITPLS)
+    from .Lee2020 import DicarloLee2020ITTissueResponseCorrelation
+    pool['dicarlo.Lee2020.IT-tissue'] = LazyLoad(DicarloLee2020ITTissueResponseCorrelation)
```

Comment:
I would call it `response_correlation` instead of `tissue`.
Likewise, I would rename the benchmark to `DicarloLee2020ITResponseCorrelation`

Reply:
yea I think this gets way too long. Maybe we can talk about it in our meeting, ideally we could find a name that is:
* short
* mentions something spatial (about the tissue)
* mentions something about the response correlation
But not sure if we can find an identifier that satisfies all three

Reply:
rename identifier to spatial_correlation and SpatialCorrelationSimilarity